scrapy/http/request/form.py

```diff
@@ -54,10 +55,15 @@ def _urlencode(seq, enc):
     return urlencode(values, doseq=1)
 
 
+def _create_parser_from_response(response, parser_cls):
```

Comment:
@kmike what do you think about promoting `Selector._root` to `Selector.root` in parsel library and use an `extended-html` parser for `lxml.html.HTMLParser` ?